phone/src/locale/en.json

```diff
@@ -115,6 +115,8 @@
     "APPS_MESSAGES_MESSAGE_GROUP_DUPLICATE": "This message group already exists",
     "APPS_MESSAGES_MESSAGE_GROUP_CREATE_FAILED": "Failed to create message group",
     "APPS_MESSAGES_MESSAGE_GROUP_CREATE_MINE": "You cannot add yourself!",
+    "APPS_MESSAGES_MESSAGE_GROUP_CREATE_ALL_NUMBERS_FAILED": "None of the phone numbers could be found!",
+    "APPS_MESSAGES_MESSAGE_GROUP_CREATE_SOME_NUMBERS_FAILED": "Some phone numbers could not be found: {{ numbers }}",
```

Comment:
```suggestion
        "APPS_MESSAGES_MESSAGE_GROUP_CREATE_SOME_NUMBERS_FAILED": "Invalid phone numbers: {{ numbers }}",
```

Reply:
Note that they are not invalid (as invalid can't even be sent) Its just that its inexistent